concordium-consensus/src/Concordium/External.hs

```diff
@@ -60,6 +60,7 @@ import Concordium.Skov (
  )
 import Concordium.TimerMonad (ThreadTimer)
 import qualified Concordium.Types.InvokeContract as InvokeContract
+import qualified Concordium.Types.Queries as TQ
```

Comment:
I think `Queries` would be more useful at call sites.

Reply:
I followed the `as Q` for other Queries we import. But since you commented it is now Queries.